docs/visual-basic/language-reference/error-messages/xml-entity-references-are-not-supported.md

```diff
@@ -1,23 +1,23 @@
 ---
 title: "XML entity references are not supported"
 ms.date: 07/20/2015
-f1_keywords: 
+f1_keywords:
   - "vbc31180"
   - "bc31180"
-helpviewer_keywords: 
+helpviewer_keywords:
   - "BC31180"
 ms.assetid: 2a393327-d8e2-4187-85b1-642b4f53b4ae
 ---
-# XML entity references are not supported
+# BC31180: XML entity references are not supported
+
+An entity reference (for example, `&copy;`) that is not defined in the XML 1.0 specification is included as a value for an XML literal. Only `&`, `"`, `<`, `>`, and `'` XML entity references are supported in XML literals.
```

Comment:
This changes the rendering. Not sure if this was intended.

Reply:
Good catch! @IEvangelist , is this something that the smart quote replacement tool might have done? It's not rendering as a copyright symbol: https://review.docs.microsoft.com/en-us/dotnet/visual-basic/language-reference/error-messages/xml-entity-references-are-not-supported?branch=pr-en-us-21061. The tool should probably not update anything in code tags.

Reply:
Yes, you're correct. The smart quotes will replace that when enabled... and it does still need to have that bug addressed where it replaces things in code that it shouldn't.